monitoring/ingestion/global-dashboard.json

```diff
@@ -0,0 +1,632 @@
+      "version": ""
+    },
+    {
+      "type": "datasource",
+      "id": "prometheus",
+      "name": "Prometheus",
+      "version": "1.0.0"
+    },
+    {
+      "type": "panel",
+      "id": "stat",
+      "name": "Stat",
+      "version": ""
+    }
+  ],
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "id": null,
+  "iteration": 1640659169017,
+  "links": [],
+  "panels": [
+    {
+      "datasource": "${DS_PROMETHEUS_SCALITY}",
+      "description": "How many instances are currently online and offline",
+      "fieldConfig": {
+        "defaults": {
+          "displayName": "Online",
+          "mappings": [],
+          "noValue": "0",
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              }
+            ]
+          }
+        },
+        "overrides": [
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Offline"
+            },
+            "properties": [
+              {
+                "id": "thresholds",
+                "value": {
+                  "mode": "absolute",
+                  "steps": [
+                    {
+                      "color": "red",
+                      "value": null
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 4,
+        "w": 12,
+        "x": 0,
+        "y": 0
+      },
+      "id": 18,
+      "options": {
+        "colorMode": "value",
+        "graphMode": "none",
+        "justifyMode": "center",
+        "orientation": "vertical",
+        "reduceOptions": {
+          "calcs": [
+            "last"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "text": {},
+        "textMode": "auto"
+      },
+      "pluginVersion": "7.5.7",
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "sum(up{job=\"${populatorJob}\"})",
+          "format": "time_series",
+          "interval": "",
+          "legendFormat": "Online",
+          "refId": "A"
+        },
+        {
+          "exemplar": true,
+          "expr": "count(up{job=~\"${populatorJob}\"})-sum(up{job=~\"${populatorJob}\"})",
+          "format": "time_series",
+          "interval": "",
+          "legendFormat": "Offline",
+          "refId": "B"
+        }
+      ],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Ingestion Producer",
+      "transformations": [],
+      "type": "stat"
+    },
+    {
+      "datasource": "${DS_PROMETHEUS_SCALITY}",
```

Comment:
Zenko-operator is replacing `${replicas}`, `${job_ingestion_processor}`, `${job_ingestion_producer}`, `${DS_PROMETHEUS}`, `${DS_LOKI}`, `${namespace}` or `${zenkoName}` with the expected value.

So, here `${DS_PROMETHEUS_SCALITY}` won't be replaced, you will need to use: `${DS_PROMETHEUS}`. 

Feel free to look at https://github.com/scality/backbeat/blob/development/8.3/monitoring/ingestion/ingestion-processor-dashboard.json to see how it is done.